log4j2/BUILD

```diff
@@ -13,12 +13,16 @@ LOG4J2_BACKEND_SRCS = glob(["src/main/java/**/*.java"])
 java_library(
     name = "log4j2_backend",
     srcs = LOG4J2_BACKEND_SRCS,
-    javacopts = ["-source 1.6 -target 1.6"],
+    javacopts = ["-source 1.8 -target 1.8"],
     tags = ["maven_coordinates=com.google.flogger:flogger-log4j2-backend:${project.version}"],
     deps = [
         "//api",
         "//api:system_backend",
+        "//grpc:grpc_context",
+        "@com_google_errorprone_error_prone_annotations",
+        "@google_bazel_common//third_party/java/guava",
```

Comment:
Probably best to avoid a Guava dependency here since the rest of Flogger avoids it. It doesn't look like you're using it for anything too significant.